docs/Fortigate-Events-Legacy-Configuration.md

```diff
@@ -0,0 +1,71 @@
+# Configuration for collection of Fortinet Fortigate logs (Preview)
+
+**This feature is part of the OMS Security and Audit solution. The solution needs to be enabled before the events can be collected.**
+
+Collection of the following 3rd party security log types is supported:
+- **Fortinet Fortigate logs FortiOS < 5.6**
+
+Fortinet Fortigate can send events in CEF starting from [FortiOS 5.6](http://help.fortinet.com/fos50hlp/56/Content/FortiOS/fortigate-whats-new/Top-LogReport-cef.htm?Highlight=syslog%20CEF), if it is not possibile to update the appliances to the new OS, this solution can be used to translate legacy Fortigate Logs in CEF and ingest them into Log Analytics.
+
+## Caveats
+
+Fortigate log parsing can have an impact on CPU utilization by the OMS agent, the solution has been tested on a dual core Virtual Machine ingesting about 40 events per second causing an increase in CPU usage of about 5%.
+
+## Configuration summary
+1. Install and onboard the OMS Agent for Linux
+2. Configure Syslog forwarding to send the required logs to the agent on UDP port 25227. This must be a dedicated syslog to be able to differentiatte >Fortigate logs from other log types
```

Comment:
why this port is hardcoded? Should not it just say use an available port for syslog?

Reply:
The port is not hardcoded, the configuration states "in this example" the same pattern used in this doc https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/docs/Security-Events-Preview-Configuration.md